### PR TITLE
PUBDEV-7387: Fixing all the problematic R examples 

### PR DESCRIPTION
**Pull request by ledell**
https://0xdata.atlassian.net/browse/PUBDEV-7387

### CODE REVIEW

**Comment by angela0xdata** [comment]
Thanks @ledell. These look good, but I get the following error when I run `R CMD check --as-cran h2o_*.*.*.*.tar.gz`:

> * checking package dependencies ... ERROR
> Packages suggested but not available:
>   ‘ggplot2’ ‘mlbench’ ‘slam’ ‘bit64’ ‘data.table’ ‘rgl’ ‘plot3Drgl’
> 
> The suggested packages are required for a complete check.
> Checking can be attempted without them by setting the environment
> variable _R_CHECK_FORCE_SUGGESTS_ to a false value.

File: h2o-r/h2o-package/R/word2vec.R [hunk 1]
```diff
@@ -25,13 +25,13 @@
 #' h2o.init()
 #' 
 #' # Import the CraigslistJobTitles dataset
-#' job.titles.path = "https://raw.githubusercontent.com/h2oai/sparkling-water/rel-1.6/examples/smalldata/craigslistJobTitles.csv"
-#' job.titles <- h2o.importFile(job.titles.path, destination_frame = "jobtitles",
+#' f <- "https://raw.githubusercontent.com/h2oai/sparkling-water/rel-1.6/examples/smalldata/"
+#' jobtitles <- h2o.importFile(pasteo(f, "craigslistJobTitles.csv"),
```
Comment (angela0xdata) [inline h2o-r/h2o-package/R/word2vec.R]: should this be paste0 instead of pasteo?

Comment (ledell) [reply h2o-r/h2o-package/R/word2vec.R]: @angela0xdata Yep, oops I thought I already changed that.  Maybe I updated that code in two places.  Thx.

Comment (ledell) [inline h2o-r/h2o-package/R/word2vec.R]:
```suggestion
#' jobtitles <- h2o.importFile(paste0(f, "craigslistJobTitles.csv"), 
```

**Comment by ledell** [comment]
Hey @angela0xdata I don't see those errors locally or on Jenkins (I think they should go away if you install those packages locally) and they're not related to these docs changes.  Everything looks good on my end so I'm going to merge.  Thanks!

File: h2o-r/h2o-package/R/frame.R [hunk 2]
```diff
@@ -4671,12 +4814,16 @@ generate_col_ind <-function(data, by) {
...
 #' library(h2o)
 #' h2o.init()
-#' air <- h2o.importFile("https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/allyears2k_headers.csv")
-#' groupCols <- c("Distance")
-#' sortCols <- c("IsArrDelayed", "IsDepDelayed")
-#' sortDirs <- c(TRUE, FALSE)
-#' h2o.rank_within_group_by(air, groupCols, sortCols, 
-#'                          sortDirs, new_col_name = "New_Rank", 
+#' 
+#' f <- "https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/allyears2k_headers.csv"
```
Comment (ledell) [inline h2o-r/h2o-package/R/frame.R]:
```suggestion
#' f <- "https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/allyears2k_headers.zip"
```

Comment (ledell) [reply h2o-r/h2o-package/R/frame.R]: This file does not exist as csv -- its a zip file.  Fixing error found in build.

**Comment by ledell** [comment]
Just fixed a small issue where the file URL did not exist:

Found on jenkins:
```
[2020-03-20T21:28:56.807Z] > f <- "https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/allyears2k_headers.csv"

[2020-03-20T21:28:56.807Z] > air <- h2o.importFile(f)

[2020-03-20T21:28:56.807Z] https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/allyears2k_headers.csv failed to importError in h2o.importFolder(path, pattern = "", destination_frame = destination_frame,  : 

[2020-03-20T21:28:56.807Z]   all files failed to import

[2020-03-20T21:28:56.807Z] Calls: h2o.importFile -> h2o.importFolder

[2020-03-20T21:28:56.807Z] Execution halted

[2020-03-20T21:28:56.807Z] * checking PDF version of manual ... OK

[2020-03-20T21:28:56.807Z] * DONE

[2020-03-20T21:28:56.807Z] 

[2020-03-20T21:28:56.807Z] Status: 1 ERROR, 1 NOTE

[2020-03-20T21:28:56.807Z] See

[2020-03-20T21:28:56.807Z]   ‘/home/jenkins/slave_dir_from_mr-0xc1/workspace/h2o-3-pipeline_PR-4438@2/r3.5-cmd-check/h2o-3/h2o-r/R/src/contrib/h2o.Rcheck/00check.log’[2020-03-20T21:28:56.807Z] > f <- "https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/allyears2k_headers.csv"

[2020-03-20T21:28:56.807Z] > air <- h2o.importFile(f)

[2020-03-20T21:28:56.807Z] https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/allyears2k_headers.csv failed to importError in h2o.importFolder(path, pattern = "", destination_frame = destination_frame,  : 

[2020-03-20T21:28:56.807Z]   all files failed to import

[2020-03-20T21:28:56.807Z] Calls: h2o.importFile -> h2o.importFolder

[2020-03-20T21:28:56.807Z] Execution halted

[2020-03-20T21:28:56.807Z] * checking PDF version of manual ... OK

[2020-03-20T21:28:56.807Z] * DONE

[2020-03-20T21:28:56.807Z] 

[2020-03-20T21:28:56.807Z] Status: 1 ERROR, 1 NOTE

[2020-03-20T21:28:56.807Z] See

[2020-03-20T21:28:56.807Z]   ‘/home/jenkins/slave_dir_from_mr-0xc1/workspace/h2o-3-pipeline_PR-4438@2/r3.5-cmd-check/h2o-3/h2o-r/R/src/contrib/h2o.Rcheck/00check.log’
```

**Pull request merged**